pkg/sdk/go/users_test.go

```diff
@@ -771,17 +776,19 @@ func TestUpdateClientIdentity(t *testing.T) {
 	}
 
 	for _, tc := range cases {
-		repoCall := pRepo.On("Evaluate", mock.Anything, mock.Anything, mock.Anything).Return(nil)
-		repoCall1 := cRepo.On("RetrieveByID", mock.Anything, mock.Anything).Return(convertClient(tc.client), tc.err)
-		repoCall2 := cRepo.On("UpdateIdentity", mock.Anything, mock.Anything).Return(convertClient(tc.response), tc.err)
+		repoCall := pRepo.On("CheckAdmin", mock.Anything, mock.Anything).Return(nil)
+		repoCall1 := pRepo.On("Evaluate", mock.Anything, mock.Anything, mock.Anything).Return(nil)
+		repoCall2 := cRepo.On("RetrieveByID", mock.Anything, mock.Anything).Return(convertClient(tc.client), tc.err)
+		repoCall3 := cRepo.On("UpdateIdentity", mock.Anything, mock.Anything).Return(convertClient(tc.response), tc.err)
```

Comment:
Let's avoid using `mock.Anything` since that does not assume anything about the actual parameters, while in real scenarios, we usually know those values. Also, let's test whether these calls are actually executed and in which order (`After`, `Before`, `Times`, `parent.AssertCalled`)...

Reply:
From the transport layer is is difficult to check the actual things pass this is same as for `parent.AssertCalled`. Would be checking the number of times it was called okay `Parent.AssertNumberOfCalls`?

Reply:
Yes, that should be sufifcient.